aws/rakefiles/kops.rake

```diff
@@ -97,5 +97,8 @@ task :display_rolling_update_cmd => [@tmpdir, :configure_kops] do
 end
 
 task :kops_rolling_update => [@tmpdir, :configure_kops] do
-  sh "KOPS_FEATURE_FLAGS='+DrainAndValidateRollingUpdate' kops rolling-update cluster #{ENV["TF_VAR_cluster_name"]} --yes"
+  wait_for("\
```

Comment:
I think we don't need to wait default 15 minutes here. Maybe, if it is unable to reach kubernetes API for a minute, there is probably something wrong with the cluster?

Reply:
Good call.